test/src/BuildTest.ts

```diff
@@ -61,6 +61,12 @@ test("version from electron-prebuilt dependency", () => assertPack("test-app-one
   }
 }))
 
+test("www as default dir", () => assertPack("test-app", allPlatformsAndCurrentArch(), {
```

Comment:
I think it is ok and better to test only current platform, not all platforms (so, our test suite will execute faster).